README.md

```diff
@@ -537,11 +537,40 @@ Available built-in strategies:
    ```markdown
    # Version 1.2.3 (2024-03-22)
 
-   - [ADDED] New feature description
-   - [FIXED] Bug fix description
+   - [Feature] New feature description
+   - [Fix] Bug fix description
```

Comment:
The code example here wouldn't result in these being all-caps, so I fixed that along with the labels (being a custom writing strategy, unless the labels were explicitly overridden, they should be the defaults).